src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -5,38 +5,47 @@
 import com.iota.iri.storage.Persistable;
 import com.iota.iri.storage.PersistenceProvider;
 import com.iota.iri.utils.Pair;
+import com.iota.iri.utils.Quietly;
 import org.apache.commons.lang3.SystemUtils;
 import org.rocksdb.*;
 import org.rocksdb.util.SizeUnit;
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.nio.file.Paths;
 import java.security.SecureRandom;
 import java.util.*;
-import java.util.concurrent.atomic.AtomicReference;
 import java.util.stream.Collectors;
 
 /**
  * Created by paul on 3/2/17 for iri.
+ * Edited by footloosejava on 03/28/2018
+ * All Sets returned are unmodifiable.
  */
 public class RocksDBPersistenceProvider implements PersistenceProvider {
 
-    private static final org.slf4j.Logger log = LoggerFactory.getLogger(RocksDBPersistenceProvider.class);
+    private static final Logger log = LoggerFactory.getLogger(RocksDBPersistenceProvider.class);
     private static final int BLOOM_FILTER_BITS_PER_KEY = 10;
 
+    private static final Pair<Indexable, Persistable> PAIR_OF_NULLS = new Pair<>(null, null);
+    private static final Set<Indexable> EMPTY_SET = Collections.emptySet();
+
     private final List<String> columnFamilyNames = Arrays.asList(
-            new String(RocksDB.DEFAULT_COLUMN_FAMILY),
```

Comment:
Btw,
Seperate commits for style changes will be much appreciated in future contributions.
Github lets me select which commits to look at for review so I can cancel noise.